### ISSUE TITLE
BLT Tests Failing in Travis

### ISSUE THREAD

**Issue by mikemadison13** [acquia/blt#1864]
My system information:
* Operating system type: Travis CI
* BLT version: 8.9.0

When I run this command:
```
blt tests:all
```

I get the following output:
```
[error]  Could not find VM. Please ensure that the VM machine name matches project.machine_name 
> tests:behat
[info] Killing all processing containing string 'runserver'...
[info] Killing all processes on port '8888'...
[info] Running server at http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[info] Waiting for response from http://127.0.0.1:8888...
[error]  Timed out.
[Mon Jul 31 19:39:15 2017] 127.0.0.1:35996 [404]: /
[Mon Jul 31 19:39:16 2017] 127.0.0.1:36000 [404]: /
[Mon Jul 31 19:39:17 2017] 127.0.0.1:36004 [404]: /
[Mon Jul 31 19:39:18 2017] 127.0.0.1:36008 [404]: /
[Mon Jul 31 19:39:19 2017] 127.0.0.1:36012 [404]: /
[Mon Jul 31 19:39:20 2017] 127.0.0.1:36016 [404]: /
[Mon Jul 31 19:39:21 2017] 127.0.0.1:36020 [404]: /
[Mon Jul 31 19:39:22 2017] 127.0.0.1:36024 [404]: /
[Mon Jul 31 19:39:23 2017] 127.0.0.1:36028 [404]: /
[Mon Jul 31 19:39:24 2017] 127.0.0.1:36032 [404]: /
[Mon Jul 31 19:39:25 2017] 127.0.0.1:36036 [404]: /
 
[error]  Command `tests:behat ` exited with code 1. 
```

See: https://travis-ci.org/Drupal4Gov/Drupal-GovCon-2017/builds/259490314?utm_source=github_status&utm_medium=notification

And I expected this to happen:
After updating to BLT 8.9.0 behat tests are failing. Nothing is jumping out at me as being wrong here?

**Comment by grasmash** [comment]
Give 8.x a try. It may output more useful info.

**Comment by mikemadison13** [comment]
this is currently blocked by https://github.com/acquia/blt/issues/1870

Note that the 8.x-dev branch may or may not help with this, but I can't get past the validate step.

**Comment by grasmash** [comment]
It looks like curl is returning a 404:
```
[Mon Jul 31 19:39:15 2017] 127.0.0.1:35996 [404]: /
```

**Comment by grasmash** [comment]
Closing because the issue appears resolved: https://travis-ci.org/Drupal4Gov/Drupal-GovCon-2017/builds/260228399?utm_source=github_status&utm_medium=notification

**Comment by nirojghale-iheartmedia** [comment]
@grasmash Is this already fixed in 8.9.2 ? because I still get this same error when I tried to run build in Travis Ci
